GRDB/Core/Database.swift

```diff
@@ -301,6 +306,20 @@ extension Database {
     }
     #endif
     
+    // For SQLite-SEE
+    #if SQLITE_HAS_CODEC && GRDB_SQLITE_SEE
+    private static func set(key: String, withEncryptionType encType: EncryptionType, forConnection sqliteConnection: SQLiteConnection) throws {
+        let prefixedKey = "\(encType.prefixValue):\(key)"
+        let data = prefixedKey.data(using: .utf8)!
+        let code = data.withUnsafeBytes {
+            sqlite3_key_v2(sqliteConnection, nil, $0.baseAddress, Int32($0.count))
```

Comment:
Regarding Swift 4.2 compatibility, it would be great to check that GRDB still compiles in Swift 4.2 (I can't do it because it don't have a SEE license). Ideally, you would check both Xcode 10.2 with SWIFT_VERSION=4.2, and Xcode 10.1. *This is not a requirement*.

**Alternatively**, you can also completely ignore Swift 4.2 compatibility, and *document* that support for SEE requires Swift 5. This is also a totally valid solution, and it would be much easier for you.

Do really as you want.